.github/workflows/test.yml

```diff
@@ -0,0 +1,22 @@
+name: Test
+on:
+  push:
+    branches-ignore:
+      - 'master'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
```

Comment:
fyi looks like that's the [default](https://github.com/actions/checkout/blob/master/action.yml#L58)

Reply:
Nice. I may just leave it in there to be explicit